src/slashCommand/slashCommand.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+import { SlashCommandBuilder } from 'discord.js';
+import { SlashCommandService } from './slashCommand.service';
+
+export interface BaseCommand {
+  slashCommandBuilder: Omit<
+    SlashCommandBuilder,
+    'addSubcommand' | 'addSubcommandGroup'
+  >;
+}
```

Comment:
A pattern I have used elsewhere is to have all the types in an interface file. Something like slashCommands.interface.ts. Might be nice to do the same here

Reply:
Good idea, done!